dislib/cluster/kmeans/base.py

```diff
@@ -105,7 +105,7 @@ def fit_predict(self, dataset):
         for subset in dataset:
             labels.append(_get_label(subset))
 
-        return np.array(compss_wait_on(labels))
+        return np.concatenate(compss_wait_on(labels))
```

Comment:
is more efficient np.concatenate than np.array? asking out of curiosity to learn.

Reply:
No, the issue was that a concatenation is necessary here.